packages/table/preview/index.tsx

```diff
@@ -94,15 +98,36 @@ const ROW_COUNTS = [
     1000000,
 ];
 
+enum CellContent {
+    EMPTY,
+    CELL_NAMES,
+    LONG_TEXT,
+};
+
+const CELL_CONTENTS = [
+    CellContent.EMPTY,
+    CellContent.CELL_NAMES,
+    CellContent.LONG_TEXT,
+];
+
 const COLUMN_COUNT_DEFAULT_INDEX = 2;
 const ROW_COUNT_DEFAULT_INDEX = 3;
 
+const ALPHANUMERIC_CHARS = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
 class MutableTable extends React.Component<{}, IMutableTableState> {
     private store = new SparseGridMutableStore<string>();
 
+    private contentGenerators = {
+        [CellContent.CELL_NAMES]: (row: number, col: number) => Utils.toBase26Alpha(col) + (row + 1),
+        [CellContent.EMPTY]: () => "",
+        [CellContent.LONG_TEXT]: () => this.generateRandomAlphanumericString(),
+    };
```

Comment:
seems like a good candidate for `const` above, no need for private member it seems. could inline `generateRandomAlphanumericString` for the third one.